l2gersync/evm_downloader_sovereign.go

```diff
@@ -134,7 +134,7 @@ func (d *downloaderSovereign) buildAppender(
 		l1InfoTreeLeaf, err := d.l1InfoTreeSync.GetInfoByGlobalExitRoot(insertGEREvent.NewGlobalExitRoot)
 		if err != nil {
 			log.Fatalf("GER %s received from L2 is not present in L1InfoTreeSync: %v",
-				insertGEREvent.NewGlobalExitRoot, err)
+				common.BytesToHash(insertGEREvent.NewGlobalExitRoot[:]).Hex(), err)
```

Comment:
If I'm not mistaken, it can be simplified.
```suggestion
				common.Hash(insertGEREvent.NewGlobalExitRoot) err)
```

Reply:
2b5b6a1cb2acb029e812f20b10e431c4380d944a